server/svix-server/tests/e2e_attempt.rs

```diff
@@ -262,6 +262,33 @@ async fn test_list_attempts_by_endpoint() {
         .unwrap();
     assert_eq!(obits_attempts.data.len(), 1);
 
+    let exploded_attempts: ListResponse<MessageAttemptOut> = client
+        .get(
+            &format!("api/v1/app/{app_id}/attempt/endpoint/{endp_id_2}/?event_types=user.exploded"),
+            StatusCode::OK,
+        )
+        .await
+        .unwrap();
+    assert_eq!(exploded_attempts.data.len(), 1);
+
+    let regular_attempts: ListResponse<MessageAttemptOut> = client
+        .get(
+            &format!("api/v1/app/{app_id}/attempt/endpoint/{endp_id_2}/?event_types[]=event.type"),
+            StatusCode::OK,
+        )
+        .await
+        .unwrap();
+    assert_eq!(regular_attempts.data.len(), 2);
+
+    let all_attempts: ListResponse<MessageAttemptOut> = client
+    .get(
+        &format!("api/v1/app/{app_id}/attempt/endpoint/{endp_id_2}/?event_types[0]=event.type&event_types[1]=user.exploded"),
+        StatusCode::OK,
+    )
+    .await
+    .unwrap();
+    assert_eq!(all_attempts.data.len(), 3);
+
```

Comment:
Slightly nitpicky, but since the way it's implemented now also permits mixing and matching the styles could you add a test for that, too?

Reply:
You mean like `?a=1&a[]=2`?

I know this implementation _technically_ allows that to work, but that's more for simplicity. I think the chance of a client mixing and matching like that is remote enough that I don't think we should commit to supporting that format.